charts/datadog/templates/cluster-agent-deployment.yaml

```diff
@@ -113,7 +113,12 @@ spec:
         {{- end }}
 {{- if or .Values.datadog.envFrom .Values.clusterAgent.envFrom }}
         envFrom:
-{{ concat .Values.datadog.envFrom .Values.clusterAgent.envFrom | toYaml | indent 10 }}
+{{- if .Values.datadog.envFrom }}
```

Comment:
An helper function can be made to avoid this repetitive code

Reply:
I'm not sure it will help that much. We can only pass one parameter to the helper. In this case it needs to the deepest `envFrom`, so we cannot handle the `.Values.datadog.envFrom` and we cannot handle the `| indent` either.

Reply:
I'm not sure it will help that much. We can only pass one parameter to the helper. In this case it needs to the deepest `envFrom`, so we cannot handle the `.Values.datadog.envFrom` and we cannot handle the `| indent` either.

Reply:
you can pass several argument by creating a temporary `dict`. but maybe it is overkill